src/herbivoresAndCarnivores.js

```diff
@@ -23,7 +23,7 @@ class Herbivore extends Animal {
 
 class Carnivore extends Animal {
   bite(animal) {
-    if (animal.hidden === false && animal instanceof Animal) {
+    if (animal.hidden === false && animal instanceof Herbivore) {
```

Comment:
Is it correct now? 
I checking if the animal is hiding and also if the animal is an instance of Herbivore

Reply:
yes, now it's ok)